test/setup.js

```diff
@@ -200,14 +208,14 @@ function extractImportantFields (parsedTorrent) {
 }
 
 function copy (pathFrom, pathTo) {
+  const cpFile = require('cp-file')
   try {
-    fs.copySync(pathFrom, pathTo)
```

Comment:
`fs.copySync` from `fs-extra` recursively copies entire directories, while `cp-file` only handles individual files, but we never need to copy directories, so it's fine.